src/main/java/org/jenkinsci/plugins/verifystatus/VerificationsPublisher.java

```diff
@@ -100,65 +109,61 @@ public String getVerifyStatusRerun() {
     return verifyStatusRerun;
   }
 
+  public Short getVerifyStatusValue() {
+    return verifyStatusValue;
+  }
+  @DataBoundSetter
+  public void setVerifyStatusValue(Short value) {
+    this.verifyStatusValue=value;
+  }
   // publish report after job completes to get correct duration
   @Override
   public boolean needsToRunAfterFinalized() {
     return true;
   }
 
-  private GerritTrigger getGerritTrigger(AbstractBuild<?, ?> build) {
-    AbstractProject<?,?> project = build.getProject();
-    Iterator<?> it = project.getTriggers().values().iterator();
-    while (it.hasNext()) {
-      Trigger<?> t = (Trigger<?>) it.next();
-      if (t.getClass().equals(GerritTrigger.class)) {
-        GerritTrigger gerritTrigger = (GerritTrigger) t;
-        return gerritTrigger;
-      }
-    }
-    return null;
+  private GerritTrigger getGerritTrigger(Run<?, ?> build) {
+    Job<?,?> project = build.getParent();
+    return (GerritTrigger) ParameterizedJobMixIn.getTrigger(project, GerritTrigger.class);
   }
 
   @Override
-  public boolean perform(AbstractBuild build, Launcher launcher,
-      BuildListener listener) throws IOException, InterruptedException {
+  public void perform(Run<?,?> build, FilePath path, Launcher launcher,
+      TaskListener listener) throws IOException, InterruptedException {
 
     Result result = build.getResult();
     // do not report results for aborted builds
     // TODO: possibly allow saving report even if build did not complete?
```

Comment:
You have changed it to report aborted reports?  I think it's more common to not report on aborts or at least it is for us because we purposely abort jobs due to a number of reasons.  An abort on purpose is not a job failure for us.   It doesn't like it's possible to set silent mode on aborts with the gerrit trigger plugin.  So I think we either need to keep 'ignore on abort' here or add it to the gerrit trigger plugin.   Another option would be to make this optional?  Maybe with a 'report aborted jobs' boolean checkbox?